### PR TITLE
env: update

### PR DESCRIPTION
**Pull request by chuanconggao**

### CODE REVIEW

**Comment by felixonmars** [comment]
:+1:

File: pages/common/env.md [hunk 1]
```diff
@@ -6,6 +6,10 @@
 
 `env`
 
+- Run a program. Often used in script after shebang (#!) for looking up the path to the program.
```
Comment (ostera) [inline pages/common/env.md]: in A script, or in scriptS, after A shebang or THE shebang.

Comment (waldyrious) [inline pages/common/env.md]: @ostera :ok_hand:

**Comment by chuanconggao** [comment]
@ostera Thanks for the comment. Just fixed it.

**Pull request merged**